src/js/sticky-header.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This file creates the sticky functionality for the header/menu so that it scrolls up and down.
+ * To make it change colors add .sticky to the _header.scss and enjoy
+ * @author Jess McClary <jess.mcclary@tricomb2b.com>
+ */
+(function ($) {
+  'use strict';
+
+  // Adding background color to header for scrolling
+  $(window).on('scroll', function(){
+    var topHeight = 100; 
```

Comment:
this needs to be outside of the scroll event handler. otherwise it gets set on every iteration and that's not really what we're after.